CMakeLists.txt

```diff
@@ -1,63 +1,86 @@
-cmake_minimum_required(VERSION 3.1)
+cmake_minimum_required(VERSION 3.5)
 project(trifinger_simulation)
 
 # Specify C++ Standard
-set(CMAKE_CXX_STANDARD 14)
+set(CMAKE_CXX_STANDARD 17)
 set(CMAKE_CXX_STANDARD_REQUIRED on)
 
 
-find_package(catkin REQUIRED COMPONENTS
-    mpi_cmake_modules
-    pybind11_catkin
-    robot_interfaces
-)
-
-find_package(Eigen3 REQUIRED)
-
-catkin_python_setup()
-
-catkin_package(
-    INCLUDE_DIRS ${EIGEN3_INCLUDE_DIR}
-)
+find_package(ament_cmake REQUIRED)
+find_package(mpi_cmake_modules REQUIRED)
+find_package(pybind11 REQUIRED)
+find_package(robot_interfaces REQUIRED)
 
-include_directories(
-    include
-    ${catkin_INCLUDE_DIRS}
-    ${EIGEN3_INCLUDE_DIR}
-)
 
+ament_python_install_package(${PROJECT_NAME} PACKAGE_DIR python/${PROJECT_NAME})
 
 
 # Python Bindings
-catkin_add_pybind11_module(drivers)
+add_pybind11_module(drivers srcpy/drivers.cpp
+    LINK_LIBRARIES
+        robot_interfaces::robot_interfaces
+)
 
 
-# build with -DBUILD_DOCUMENTATION=ON to build documentation
-build_doxygen_documentation()
+add_doxygen_documentation()
+
+
+# Installation
+install(DIRECTORY include/${PROJECT_NAME}/
+        DESTINATION include/${PROJECT_NAME})
+
+install(
+    PROGRAMS
+        demos/catkin/demo_object_tracker.py
+        demos/catkin/demo_robot_interface.py
+        demos/demo_cameras.py
+        demos/demo_control.py
+        demos/demo_inverse_kinematics.py
+        demos/demo_load_gym_env.py
+        demos/demo_plain_torque_control.py
+        demos/demo_random_policy.py
+        demos/demo_trifinger_platform.py
+
+        scripts/check_position_control_accuracy.py
+        scripts/evaluate_policy.py
+        scripts/profiling.py
+        scripts/pybullet_backend.py
+        scripts/replay_action_log.py
+        scripts/rrc_evaluate
+        scripts/run_evaluate_policy_all_levels.py
+        scripts/run_replay_all_levels.py
+    DESTINATION lib/${PROJECT_NAME}
+)
```

Comment:
idem install files and rename + add exec

Reply:
done